src/main/java/com/blackducksoftware/integration/fortify/batch/util/HubServices.java

```diff
@@ -169,9 +170,9 @@ private String getVulnerabililtyBomComponentUrl(final ProjectVersionView project
      */
     private List<VulnerableComponentView> getVulnerabililtyComponentViews(final String vulnerabililtyBomComponentUrl) throws IntegrationException {
         logger.info("Getting Hub Vulnerability info");
-        final HubResponseService hubResponseService = hubServicesFactory.createHubResponseService();
+        final HubService hubResponseService = hubServicesFactory.createHubService();
         HubPagedRequest hubPagedRequest = hubResponseService.getHubRequestFactory().createPagedRequest(500, vulnerabililtyBomComponentUrl);
-        return hubResponseService.getAllItems(hubPagedRequest, VulnerableComponentView.class);
+        return hubResponseService.getAllViews(hubPagedRequest, VulnerableComponentView.class, "application/vnd.blackducksoftware.list-1+json");
```

Comment:
Not related directly to this PR, but I'm wondering if you know what the versioning schema of these hub common class are. Right now the class 'VulnerableComponentView' is the representation of the v1 media type for the vulnerable components API. Will this class change if the v2 media type is available or will there be a different class for the v1 media type?

I'm wondering because at any point - since you've updated the gradle file to always get the latest version of hub-common you'll likely get some set of unexpected changes which could lead to runtime issues if VulnerableComponentView changes to match the v2 media type if it becomes available.

Perhaps not urgent, but something work asking about to ensure that you know what to expect as these APIs are versioned.

Reply:
We tried modifying the hub-common to support vulnerability api based on media types but the Integrations team is not ready to approve the PR related to the media types in hub-common. In future, if there is any change in media type, we need to manually modify the respective codes, test it and ship it as next release.

Reply:
Thats not quite what I mean. I'm talking more about the code for VulnerableComponentView. It seems thats tied to the latest media type. You'll want to keep using the old media type until you're ready to update. There will need to be support for this in hub-common.

Reply:
Sure Rob. I think, they might have different version of 'VulnerableComponentView' for different media types.